pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1500,6 +1500,15 @@
       ?~  channel
         :_  state  :_  ~
         [duct %pass /flog %d %flog %crud %eyre-no-channel >id=channel-id< ~]
+      ::  it's possible that this is a fact emitted directly alongside a fact
+      ::  that triggered a clog & closed the subscription. in that case, just
+      ::  drop the fact.
+      ::
+      ?:  ?&  ?=(%fact -.sign)
+              !(~(has by subscriptions.u.channel) request-id)
+          ==
+        ~&  [%e %fact-without-subscription channel-id request-id]
```

Comment:
I suspect this might be too noisy in practice. Also, what could the user actually do in response to this printf?

Reply:
My thinking was, this should only happen directly after an `%e %clog` printf, and stand out as suspicious when they show up on their own. But you're right it's more worrying than informative, and could become too loud.